nebullvm/operations/optimizations/compilers/tensor_rt.py

```diff
@@ -323,19 +328,39 @@ def execute(
                     subprocess.run(cmd, stdout=subprocess.DEVNULL)
 
                 # First try with simplified model
-                onnx_model_path = simplified_model
-                assert os.path.isfile(onnx_model_path)
+                self.onnx_model_path = simplified_model
+                assert os.path.isfile(self.onnx_model_path)
             except Exception:
                 # Use original model
                 self.logger.warning(
                     "Unable to simplify model with ONNX Simplifier. "
                     "Original ONNX model will be used to build "
                     "TensorRT engine"
                 )
-                onnx_model_path = str(model)
+                self.onnx_model_path = str(model)
                 self.simplify_model = False
-        else:
-            onnx_model_path = str(model)
+        elif self.onnx_model_path is None:
+            self.onnx_model_path = str(model)
+
```

Comment:
No because it enters in the if or in the elif only at the first call of the execute() (for the none quantization_type). For Half and STATIC, it must skip those statements